lib/components/inputs/UploadField/UploadField.tsx

```diff
@@ -14,11 +15,14 @@ interface UploadFieldProps {
   onReadError?: (error?: Error) => void
   wrapperClass?: string
   placeholder?: string
-  label: string
+  label?: string | ReactElement
```

Comment:
I guess there should be ReactNode, because we don't render these props as <label and <description 
```suggestion
  label?: ReactNode
```

Reply:
Good point. But ReactNode has ReactElement inside it, so probably not what we want to here. Plus is how `label` defined in other input components (eg. TextBox, TextArea). But in this case `label` should stay string and `description` could be ReactNode.